assets/javascripts/discourse/initializers/discourse-automation.js

```diff
@@ -3,8 +3,58 @@ import { ajax } from "discourse/lib/ajax";
 import { makeArray } from "discourse-common/lib/helpers";
 import { withPluginApi } from "discourse/lib/plugin-api";
 
+let _btnClickHandlers = {};
+
+function _handleEvent(event) {
```

Comment:
We probably should use a better name. This is not just "some" event, it's the "lastCheckedBy" event.